### DISCUSSION TITLE
Integration with supercharged links

### DISCUSSION THREAD

**Discussion by acepelon** [oxdc/obsidian-vertical-tabs#79] [Ideas]
Hi there and thank you for all your hard work.  I use a lot of tabs and managing them has been a chore.

I use Supercharged links/ Style Settings to add color and emojis to my tabs dynamically based on metadata.  Without your plugin, that allowed me to scan the horizontal list of tabs to locate a bit more easily tabs with specific emojis.  It would be great if that was supported with vertical tabs!

Here's an example of what the emoji's and tab colors look like without vertical tabs
![CleanShot 2024-08-16 at 09 57 15](https://github.com/user-attachments/assets/668d6613-419c-458b-894e-806b623ae41e)

Thanks!
ACE

**Comment by oxdc** [comment]
Thanks for your suggestion! It is a very good idea! I'll add this feature after some bug fixes and refactoring the session management. Stay tuned. 🎉

**Comment by oxdc** [comment]
Hi @acepelon @zanodor @fornewfork, and @flowing-abyss ! Thank you for your patience and support! I previously mentioned plans to add support for Supercharged Links after addressing some bug fixes and refactoring session management. However, due to time constraints and the complexity of maintaining compatibility with multiple plugins, I’ve decided to prioritize core improvements and defer this integration.

For more details, please refer to #113. I hope you understand, and I truly appreciate your continued support and feedback!

**Comment by rodrigo-rufino** [comment]
Any updates on this matter? How is it in the project's prioritization? 👀
